x-pack/plugins/security_solution/server/lib/detection_engine/notifications/schedule_notification_response_actions.ts

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { map, uniq } from 'lodash';
+import type { SetupPlugins } from '../../../plugin_contract';
+
+interface OsqueryQuery {
+  id: string;
+  query: string;
+  ecs_mapping: Record<string, Record<'field', string>>;
+  version: string;
+  platform: string;
+}
+
+interface OsqueryResponseAction {
+  actionTypeId: '.osquery';
+  params: {
+    id: string;
+    queries: OsqueryQuery[];
+    savedQueryId: string;
+    query: string;
+    packId: string;
+    ecs_mapping?: Record<string, { field?: string; value?: string }>;
+  };
+}
+
+export interface ResponseAction {
+  actionTypeId: string;
+  params: { [p: string]: unknown };
+}
```

Comment:
This type is already defined in another place

Reply:
removed 👍